src/reporters/console/console-reporter.js

```diff
@@ -27,6 +27,12 @@ const tty = require('tty');
 type Row = Array<string>;
 type InquirerResponses<K, T> = {[key: K]: Array<T>};
 
+// fixes bold on windows
+// $FlowFixMe TERM is not a string?
+if (process.platform === 'win32' && !/^xterm/i.test(process.env.TERM)) {
```

Comment:
I can test it, this was copied from /chalk/chalk/blob/v1.1.1/index.js#L8

startsWith is not case insensitive.